server/routes/list.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const router = express.Router();
+const protect = require('../middleware/auth');
+const { get, getAll, create, deleteItem, update } = require('../controllers/list')
+
+//create
+router.route('/').post(protect, (req, res) => create(req, res));
```

Comment:
```suggestion
router.route('/').post(protect, create);
```
You can simplify this by passing the controller like so